src/custom-element.js

```diff
@@ -566,6 +566,11 @@ function createBaseCustomElementClass(win) {
 
       /** @private {?./layout-delay-meter.LayoutDelayMeter} */
       this.layoutDelayMeter_ = null;
+
+      if (this.whenUpgradeToCustomElement_) {
```

Comment:
changed to `__AMP_UPGRADE_CUS_ELE_PROMISE__` like what we did in `action-imp.js`